src/core/client/stream/test/fixtures.ts

```diff
@@ -869,6 +870,49 @@ export const storyWithDeepestReplies = denormalizeStory(
   )
 );
 
+export const replyableComment: GQLComment = {
+  ...comments[0],
+  canReply: true,
```

Comment:
that field is set to true on the base comment, but not necessarily so. that array is just an array of comments, not "comments that can be replied to". I prefer to make things explicit rather than implicit to avoid tight coupling (I dont know why someone would change the values in that array, but if they made some valid change to one of them and a bunch of tests broke, I'd call that tight coupling)